data_steward/utils/participant_summary_requests.py

```diff
@@ -41,7 +56,7 @@ def get_access_token():
 
 def get_participant_data(url, headers):
     """
-    Fetches participant data via ParticipantSummary API
+    Fetches participant data via ParticipantSummary API with a retrieval limit of 3 minutes
```

Comment:
I'm not sure I'd add the bit about the 3 minutes to this comment.  There's nothing inherently limiting this to 3 minutes via the built in URLS.

Reply:
Forgot to remove that from when there was a time limit 🙈